build.sbt

```diff
@@ -2,19 +2,23 @@ name := "data-io"
 
 organization := "com.zeotap"
 
-scalaVersion := "2.12.15"
-version := "2.1.0"
+scalaVersion := "2.11.12"
+version := "1.2.0"
 
 import ReleaseTransformations._
 
-val sparkVersion = "3.3.1"
+val sparkVersion = "2.4.3"
 val beamVersion = "2.33.0"
```

Comment:
I think we should keep a separate base branch for Spark 2. If there is a requirement for both Spark 2 and Spark 3 at the same time which is the case usually, this change would hinder that

Reply:
Created a new base branch named **zeotap-development-spark2** taking **zeotap-development** as its base for all spark 2 related modifications